srtcore/api.cpp

```diff
@@ -257,8 +260,15 @@ int CUDTUnited::startup()
 
    m_bClosing = false;
 
-   setupMutex(m_GCStopLock, "GCStop");
-   setupCond(m_GCStopCond, "GCStop");
+   try
+   {
+       setupMutex(m_GCStopLock, "GCStop");
+       setupCond(m_GCStopCond, "GCStop");
+   }
+   catch (...)
```

Comment:
`catch (...)` is not very good. Which exactly exception can be thrown here?

Reply:
Those referring to the mutex could be various different exceptions, also platform-dependent.